wiki/markdown/docker.md

```diff
@@ -0,0 +1,61 @@
+# Docker
+
+In Eiffel-Intelligence source code a Dockerfile is provided which helps the developer or user to build the local Eiffel-Intellegence source code changes to a Docker image.
```

Comment:
I think it sounds more natural to have "source code repository"

Reply:
Fixed.